docs/source/index.rst

```diff
@@ -19,6 +19,7 @@ Contents
    :maxdepth: 1
    :titlesonly:
 
+   user_guide
    installation
```

Comment:
Should we add external resources here?

At the moment it's not easy to find `external resources` page

Reply:
I don't think that this page is that necessary to be in the header.

Reply:
He opens "User Guide" section and it will be a page inside this section.